From 5991e8b0acf1dbe3487c1f246a1d215b346a61f0 Mon Sep 17 00:00:00 2001
From: Yann Dirson <ydirson@altern.org>
Date: Sat, 17 Jun 2006 00:03:41 +0200
Subject: [PATCH] Separate CPPFLAGS from CFLAGS

This allows to set CGLAGS on command-line without interfering with
preprocessor options
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2e34ec6..05ca856 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 MAJOR=2
 MINOR=1
 CC?=gcc
-CFLAGS?=-g -O2 -Wall 
-CFLAGS+=-I. -DVERSION=\"$(MAJOR).$(MINOR)\"
+CFLAGS?=-g -O2 -Wall
+CPPFLAGS?=-I. -DVERSION=\"$(MAJOR).$(MINOR)\"
 prefix?=/usr/local
 OBJS=\
 	cbtcommon/debug.o\
-- 
1.6.5.GIT

